configs/s2ef/200k/schnet/schnet.yml

```diff
@@ -17,10 +17,10 @@ optim:
   lr_initial: 0.0005
   lr_gamma: 0.1
   lr_milestones: # epochs at which lr_initial <- lr_initial * lr_gamma
-    - 5
-    - 8
-    - 10
-  warmup_epochs: 3
+    - 4000
+    - 6000
+    - 8000
+  warmup_epochs: 2000
```

Comment:
Should we rename this to `warmup_iters` instead?

Reply:
Will do.